index.test-d.ts

```diff
@@ -1,15 +1,17 @@
 import {expectType, expectError} from 'tsd';
 import {includeKeys, excludeKeys} from './index.js';
 
+const propSymbol = Symbol('test');
```

Comment:
```suggestion
const propertySymbol = Symbol('test');
```

Reply:
Done in c4552a01e5919f9758d0cbe4080dbd4ae1b1efc0